scripts/run.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 export LD_LIBRARY_PATH=/libmesos-bundle/lib:/libmesos-bundle/lib/mesos:$LD_LIBRARY_PATH
-export JENKINS_SLAVE_AGENT_PORT=$PORT_AGENT
+# Set JENKINS_SLAVE_AGENT_PORT to the value of networking.agent-port from universe/config.json
+# with requirePorts:true set in the marathon runspec to ensure the port is available.
+export JENKINS_SLAVE_AGENT_PORT=$PORT3
```

Comment:
Does `java .. -jar ${JENKINS_FOLDER}/jenkins.war` actually use this environment variable?

From my observations and from the code in https://github.com/jenkinsci/docker/blob/7d1ab73338e76b49c856e07a63550d80eeb35bf3/jenkins.sh#L21 it looks like it doesn't; 
shouldn't we pass the port as an option `-Djenkins.model.Jenkins.slaveAgentPort=...`?

Reply:
Good catch!